deploy/example_clusters/cr-cluster-training-engine.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: "kubedirector.hpe.com/v1beta1"
+kind: "KubeDirectorCluster"
+metadata: 
+  name: "training-engine"
```

Comment:
Again, since app is called training-engine let us call this "training-environment"